### PR TITLE
page-form - create page

### PR DESCRIPTION
**Pull request by kand**

### CODE REVIEW

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;pcdMiXNQ&#x2F;53-25-create-dynamic-page-form-directive"> (.25) create dynamic page form directive</a>

File: app/components/dynamic-content/dynamic-content-form/dynamic-content-form.tests.js [hunk 1]
```diff
@@ -21,11 +21,11 @@ describe('Directive: dynamicContentForm', function () {
         window.testHelper.directiveMock($compileProvider, 'dynamicContentFormFieldMock');
         window.testHelper.directiveMock($compileProvider, 'dynamicContentFormFieldText');
 
-        var key = 'DIRECTIVE_NAMES_MAP';
-        var mapCopy = angular.copy($injector.get(key));
-        mapCopy[mockDirectiveNameKey] = mockDirectiveName;
+        //var key = 'DIRECTIVE_NAMES_MAP';
```
Comment (collin) [inline app/components/dynamic-content/dynamic-content-form/dynamic-content-form.tests.js]: Wasn't sure what to do with these mocks.

File: app/components/dynamic-content/dynamic-content-form/dynamic-content-form-object/dynamic-content-form-object.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  .directive('dynamicContentFormObject', [
+    '_', '$compile', 'DIRECTIVE_NAMES_MAP',
+    function (_, $compile, DIRECTIVE_NAMES_MAP) {
+      var DynamicContentFormObjectError = BulbsCmsError.build('<dynamic-content-form-object>');
+
+      return {
+        link: function (scope, element, attrs) {
+          var $form = element.find('form');
+
+          scope.$watch('schema', function () {
```
Comment (collin) [inline app/components/dynamic-content/dynamic-content-form/dynamic-content-form-object/dynamic-content-form-object.js]: Do we need to `$watch` the schema?

This is now passed directly to the directive scope and won't be changing.

**Comment by kand** [comment]
Basics in place, merging into base branch.

**Pull request merged**